galley/pkg/server/server.go

```diff
@@ -108,6 +110,9 @@ func newServer(a *Args, p patchTable) (*Server, error) {
 		if err != nil {
 			return nil, err
 		}
+		if err := p.createResourceTypes(k); err != nil {
```

Comment:
It would be good to guard CRD creation with a flag of some sort. (1) for legacy installs where CRDs are managed by some other means and (2) local testing where we don't necessarily need/want galley to modify the cluster.

Reply:
The magic here is that we are going to delete CRDs from Helm's database.  As this code is written today, a CRD creation fails if one exists already.  In scenario 1 above, the `tiller` deletion of CRDs (which leaves Kubernetes CRDs unaffected) provides an upgrade path for legacy installs.  For scenario 2, we could offer a galley flag as you mentioned.